Tests/Testo/Invoice/Setting/QuoteSalesOrderInvResetServiceTest.php

```
@@ -0,0 +1,142 @@
+    /**
+     * @return array{0: DatabaseInterface&m\MockInterface, 1: SchemaProviderInterface&m\MockInterface,
+     *     2: DatabaseBackupService&m\MockInterface, 3: Aliases&m\MockInterface}
+     */
+    private function makeDeps(): array
+    {
+        /** @var DatabaseInterface&m\MockInterface $database */
+        $database = m::mock(DatabaseInterface::class);
+        /** @var SchemaProviderInterface&m\MockInterface $schemaProvider */
+        $schemaProvider = m::mock(SchemaProviderInterface::class);
+        /** @var DatabaseBackupService&m\MockInterface $backupService */
+        $backupService = m::mock(DatabaseBackupService::class);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Mockery' in method 'makeDeps'. (undefined)

(StaticAccess)